src/addRemove.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-cycle */
+import { list, saveChanges, saveLocalstorage } from './drag.js';
+import { todoList, del } from './index.js';
+
+export default function addTask(task) {
+  todoList.innerHTML = '';
+  const newTask = document.createElement('div');
+  newTask.classList.add('mini-section');
+  newTask.classList.add('item');
+  newTask.setAttribute('draggable', 'true');
+  newTask.innerHTML = `
+      <span>
+      <input class='check' type='checkbox' id='task-description' name='task-description' value='${task}'>${task}
+      <i class="fas fa-ellipsis-v ellipsis" style="color: gray; float: right;"></i>
+      </span>
+      `;
+  todoList.appendChild(newTask);
+  saveChanges();
+  saveLocalstorage(list);
+  document.location.reload(true);
+}
```

Comment:
* Currently only one task can be added in the `list` array,  so as a result other features[[These features](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new,in%20local%20storage.)] of this milestone cant be tested kindly fix this 🙏